src/shell/js/composeui-fdc3/src/ComposeUIDesktopAgent.ts

```diff
@@ -102,14 +102,15 @@ export class ComposeUIDesktopAgent implements DesktopAgent {
                 reject(new Error("The current channel have not been set."));
                 return;
             }
-            if (typeof contextType != 'string' || !contextType) {
-                reject(new Error("The contextType was type of ContextHandler, which would use a deprecated function, please use string or null for contextType!"));
+            if (typeof contextType == 'function') {
+                handler = contextType;
+                contextType = null;
                 return;
```

Comment:
I don't think you want to return here, it won't add any listeners.

Reply:
Yep, I already fixed this in local, I moved the PR back to draft because I noticed some strange behaviour, and broke some more stuff after the initial publish of the PR

Reply:
Except the part where Github decided to not move it back to draft apparently